WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -330,6 +316,13 @@ on_retreat = {
 
 #Chinese traits
 son_of_heaven = {
+	potential = {
+		OR = {
+			culture_group = sinic
+			religion = confucian
+		}
+		higher_real_tier_than = KING
+	}
```

Comment:
No it isn't. It's very simple:

1. You must be Confucian or a Chinaman.
2. You must be an Emperor of some kind.

I plan on implementing Events and Modifiers representing the Mandate of Heaven later on, using this Trait.